pkg/api/v20170131/validate.go

```diff
@@ -98,6 +99,7 @@ func (a *Properties) Validate() error {
 		if agentPoolProfile.OSType == Windows {
 			switch a.OrchestratorProfile.OrchestratorType {
 			case Swarm:
+			case SwarmMode:
```

Comment:
I thought we were disabling support for SwarmMode Windows since the template is not ready.